tiledb/sm/cpp_api/fragment_info.h

```diff
@@ -177,6 +177,104 @@ class FragmentInfo {
     return std::make_pair(start, end);
   }
 
+  /** Returns the number of MBRs in the fragment with the given index. */
+  uint64_t mbr_num(uint32_t fid) const {
+    auto& ctx = ctx_.get();
+    uint64_t ret;
+    ctx.handle_error(tiledb_fragment_info_get_mbr_num(
+        ctx.ptr().get(), fragment_info_.get(), fid, &ret));
+    return ret;
+  }
+
+  /**
+   * Retrieves the MBR of the fragment with the given index on the given
+   * dimension index.
+   */
+  void get_mbr(uint32_t fid, uint32_t mid, uint32_t did, void* mbr) const {
+    auto& ctx = ctx_.get();
+    ctx.handle_error(tiledb_fragment_info_get_mbr_from_index(
+        ctx.ptr().get(), fragment_info_.get(), fid, mid, did, mbr));
+  }
+
+  /**
+   * Retrieves the MBR of the fragment with the given index on the given
+   * dimension name.
+   */
+  void get_mbr(
+      uint32_t fid,
+      uint32_t mid,
+      const std::string& dim_name,
+      void* mbr) const {
+    auto& ctx = ctx_.get();
+    ctx.handle_error(tiledb_fragment_info_get_mbr_from_name(
+        ctx.ptr().get(),
+        fragment_info_.get(),
+        fid,
+        mid,
+        dim_name.c_str(),
+        mbr));
+  }
+
+  /**
+   * Returns the MBR of the fragment with the given index on the given
+   * dimension index. Applicable to string dimensions.
+   */
+  std::pair<std::string, std::string> mbr_var(
+      uint32_t fid, uint32_t mid, uint32_t did) const {
+    auto& ctx = ctx_.get();
+    uint64_t start_size, end_size;
+    std::string start, end;
+    ctx.handle_error(tiledb_fragment_info_get_mbr_var_size_from_index(
+        ctx.ptr().get(),
+        fragment_info_.get(),
+        fid,
+        mid,
+        did,
+        &start_size,
+        &end_size));
+    start.resize(start_size);
+    end.resize(end_size);
+    ctx.handle_error(tiledb_fragment_info_get_mbr_var_from_index(
+        ctx.ptr().get(),
+        fragment_info_.get(),
+        fid,
+        mid,
+        did,
+        &start[0],
+        &end[0]));
+    return std::make_pair(start, end);
+  }
+
+  /**
+   * Returns the MBR of the fragment with the given index on the given
+   * dimension name. Applicable to string dimensions.
+   */
+  std::pair<std::string, std::string> mbr_var(
```

Comment:
Call it `mbr()`? We usually use `get_*` if we pass the return by pointer to the function arguments.

Reply:
I can change this, but I named them to be consistent with `get_non_empty_domain` and `non_empty_domain_var` which also respectively return `void` and `std::pair<std::string, std::string>`

Reply:
Yeah, probably that was an omission. I think we should stick to having `get_*` accept the return values as arguments with pointers.